open-metadata-test/open-metadata-fvt/access-services-fvt/community-profile-fvt/pom.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>access-services-fvt</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>2.10-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <scm>
+        <connection>scm:git:git://github.com/odpi/egeria.git</connection>
+        <developerConnection>scm:git:ssh://github.com/odpi/egeria.git</developerConnection>
+        <url>http://github.com/odpi/egeria/tree/master</url>
+    </scm>
+
+    <artifactId>community-profile-fvt</artifactId>
+    <name>Community Profile OMAS FVT</name>
+    <description>
+        Community Profile Open Metadata Access Service (OMAS) FVT Suite.
+    </description>
+
+    <!-- TODO: Properties used for FVT tests. Note tests currently hardcoded, whilst exec plugin is not. Needs consolidation -->
+    <properties>
+        <fvt.serverport>10446</fvt.serverport>
```

Comment:
Data manager fvt uses this port. We should try to keep unique to avoid issues with parallelism